src/multiparser.js

```diff
@@ -163,6 +165,10 @@ function getText(options, node) {
   return options.originalText.slice(util.locStart(node), util.locEnd(node));
 }
 
+function deIndent(options, doc) {
+  return align(-options.tabWidth, doc);
```

Comment:
I've been trying to avoid doing that. Could you instead not indent when you are about to print them like you do for html?